pkg/storageincentives/proof.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package storageincentives
+
+import (
+	"github.com/ethersphere/bee/pkg/cac"
+	storer "github.com/ethersphere/bee/pkg/storer"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+func sampleChunk(items []storer.SampleItem) (swarm.Chunk, error) {
```

Comment:
Would be nice to have a general description of the `sampleChunk` and `sampleHash` functions even they are not exported.